ashapp/trajectory_generators.py

```diff
@@ -42,9 +43,25 @@ def generate_traj_from_obsdf(csvname):
     #return obsdf
     outp = {}
     # one trajectory run per time period.
-    timelist = obsdf['time'].unique()
-    for time in timelist:
-        newdf = obsdf[obsdf['time']==time]
+    """
+    This block creates tdump files grouped in 
+    same measured time
+    """ 
+#    timelist = obsdf['time'].unique()
+#    for time in timelist:
+#        newdf = obsdf[obsdf['time']==time]
+#        time = pd.to_datetime(time)
+#        yield time, generate_traj_from_df(newdf)
+
+    """
+    This block creates single tdump file for
```

Comment:
may want to create a new function so have one function which creates single trajectory and one which retains original functionality of having multiple trajectories in single file.